### PR TITLE
Update default runtime to node20

### PR DESCRIPTION
**Pull request by takost**
Node12 was deleted from runner. Node20 was added to Actions Runner on v2.308.0.
Node16 has en end of life on 11 Sep 2023.

This PR updates the default runtime to node20, rather then node16

### CODE REVIEW

**Review by Spiritaine — APPROVED** [review]

**Review by Link- — APPROVED** [review]
🚢

**Review by 04131988 — APPROVED** [review]

**Comment by zkochan** [comment]
It looks like this change was considered a breaking change, right? May I know why? Isn't this only changing the runtime of the `checkout` action? Hence, the user's code will still run on the node.js runtime configure by the user.

File: action.yml [hunk 1]
```diff
@@ -87,6 +87,6 @@ inputs:
     description: The base URL for the GitHub instance that you are trying to clone from, will use environment defaults to fetch from the same instance that the workflow is running from unless specified. Example URLs are https://github.com or https://my-ghes-server.example.com
     required: false
```
Comment (HUFF-ANDERSON) [inline action.yml:88]: Exe.run

**Review by 920506837 — COMMENTED** [review]
takost/update-node20

**Comment by kikeavril16** [comment]
Alguien me explica que es esta app y que son los códigos y las solicitudes de extracción?? Porque estoy aquí y ni se para que sirve esto

**Comment by 920506837** [comment]
1436

**Pull request merged**